packages/scripts/webapp/preset/config/.eslintrc

```diff
@@ -3,16 +3,33 @@
   "parser": "babel-eslint",
   "rules": {
     "arrow-parens": [2, "as-needed"],
-    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.js", "**/*.spec.js"]}],
+    "comma-dangle": ["error", "only-multiline"],
+    "function-paren-newline": 0,
+    "implicit-arrow-linebreak": 0,
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.js", "**/*.spec.js"], "packageDir": ["./", "./node_modules/@talend/scripts/"]}],
     "indent": ["error", "tab", { "SwitchCase": 1 }],
+    "jsx-a11y/label-has-associated-control": 2,
+    "jsx-a11y/label-has-for": 0, // deprecated and buggy, replaced by jsx-a11y/label-has-associated-control
+    "no-else-return": [2, { "allowElseIf": true }],
     "no-mixed-operators": "off",
+    "no-restricted-globals": 0,
+    "no-restricted-properties": 0,
     "no-tabs": "off",
-    "react/prefer-es6-class": 0,
+    "object-curly-newline": 0,
+    "operator-linebreak": 0,
+    "prefer-destructuring": 0,
+    "react/button-has-type": 0,
+    "react/destructuring-assignment": 0,
+    "react/forbid-prop-types": 0,
+    "react/jsx-filename-extension": [1, { "extensions": [".js"] }],
     "react/jsx-indent": ["error", "tab"],
     "react/jsx-indent-props": ["error", "tab"],
-    "react/jsx-filename-extension": [1, { "extensions": [".js"] }],
+    "react/jsx-one-expression-per-line": 0,
+    "react/no-array-index-key": 0,
+    "react/no-this-in-sfc": 0, // because of ref affectation
     "react/no-unused-prop-types": [2, { "skipShapeProps": true }],
-    "react/forbid-prop-types": [0]
```

Comment:
Should the `tools-javascript` project be updated (or deprecated) ?
https://github.com/Talend/tools/blob/master/tools-javascript/.eslintrc

Reply:
In fact I'm not that sure we should merge this PR.
At least we have to determine a path, because this will impact a lot the current code. So it's in stand by now.
I think we should deprecate the one in tools, and talend/scripts should be the only source of truth

Reply:
Yes lets drop Talend/tools.

Reply:
PR to remove it : https://github.com/Talend/tools/pull/79